src/main/java/com/gregtechceu/gtceu/client/renderer/machine/ChargerRenderer.java

```diff
@@ -56,24 +56,26 @@ public void renderMachine(List<BakedQuad> quads, MachineDefinition definition, @
         if (side == frontFacing && modelFacing != null) {
             var bakedFaces = new ArrayList<BakedQuad>();
             switch (state) {
-                case IDLE -> bakedFaces.add(FaceQuad.bakeFace(modelFacing, ModelFactory.getBlockSprite(CHARGER_IDLE),
-                        modelState, -1, 0, false, true));
+                case IDLE -> bakedFaces.add(StaticFaceBakery.bakeFace(modelFacing,
+                        ModelFactory.getBlockSprite(CHARGER_IDLE), modelState, -1, 0, false, true));
                 case RUNNING -> {
-                    bakedFaces.add(FaceQuad.bakeFace(modelFacing, ModelFactory.getBlockSprite(CHARGER_RUNNING),
-                            modelState, -1, 0, true, true));
+                    bakedFaces.add(StaticFaceBakery.bakeFace(modelFacing,
+                            ModelFactory.getBlockSprite(CHARGER_RUNNING), modelState, -1, 0, true, true));
                     if (ConfigHolder.INSTANCE.client.machinesEmissiveTextures) {
                         bakedFaces.add(
-                                FaceQuad.bakeFace(modelFacing, ModelFactory.getBlockSprite(CHARGER_RUNNING_EMISSIVE),
-                                        modelState, -101, 15, true, false));
+                                StaticFaceBakery.bakeFace(modelFacing,
+                                        ModelFactory.getBlockSprite(CHARGER_RUNNING_EMISSIVE), modelState, -101, 15,
+                                        true, false));
                     }
                 }
                 case FINISHED -> {
-                    bakedFaces.add(FaceQuad.bakeFace(modelFacing, ModelFactory.getBlockSprite(CHARGER_FINISHED),
+                    bakedFaces.add(StaticFaceBakery.bakeFace(modelFacing, ModelFactory.getBlockSprite(CHARGER_FINISHED),
                             modelState, -1, 0, true, true));
                     if (ConfigHolder.INSTANCE.client.machinesEmissiveTextures) {
                         bakedFaces.add(
-                                FaceQuad.bakeFace(modelFacing, ModelFactory.getBlockSprite(CHARGER_FINISHED_EMISSIVE),
-                                        modelState, -101, 15, true, false));
+                                StaticFaceBakery.bakeFace(modelFacing,
+                                        ModelFactory.getBlockSprite(CHARGER_FINISHED_EMISSIVE), modelState, -101, 15,
+                                        true, false));
```

Comment:
I can't add it as a suggestion, but please consider reformatting this as follows:

```java
                        bakedFaces.add(StaticFaceBakery.bakeFace(
                                modelFacing, ModelFactory.getBlockSprite(CHARGER_FINISHED_EMISSIVE), 
                                modelState, -101, 15, true, false
                        ));
```